docs/tpm_renders.md

```diff
@@ -0,0 +1,23 @@
+## The TPM concept
+According to the many varied pinouts necessary was design the concept of TPM 
+module which can fit the motherboard pinout. There was the consideration of 
+two kinds of the interface (SPI and LPC). The whole setup would be made of two 
+PCBs. One is the main PCB with the MCU and SPI or LPC configured socket and the 
+second is the adapter that fits the motherboards pinouts.
+![Flash](images/renders/concept_tpm.svg)
+
```

Comment:
please remove the empty line. leave only one

Reply:
Empty line removed in  commit `28fb95abb7080013df7b8d30a6991f7dfabd18a4` . This commit also deletes html tags used for images.